src/Sentry.Compiler.Extensions/Sentry.Compiler.Extensions.csproj

```diff
@@ -15,7 +15,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.0" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" PrivateAssets="all"/>
```

Comment:
note: [Microsoft.CodeAnalysis.CSharp v4.3.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/4.3.0) is unlisted

switching to the next patch release